Client/Frontend/Browser/Tab.swift

```diff
@@ -275,7 +275,7 @@ class Tab: NSObject {
     self.configuration = configuration
     rewardsId = UInt32.random(in: 1...UInt32.max)
     nightMode = Preferences.General.nightModeEnabled.value
-    syncTab = tabGeneratorAPI?.createBraveSyncTab()
+    syncTab = tabGeneratorAPI?.createBraveSyncTab(isOffTheRecord: true)
```

Comment:
@soner-yuksel can you check if passing `true` is correct?. it's new in 1.47.173

Reply:
Yes Nuo this is correct, thank you. The tabs that are synced should be off the record but it reminded I need to test Open Tabs functionality after @Brandon-T's changes.